src/constants/fallback-trace-name.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default '<cannot-find-trace-name>';
```

Comment:
will `<>` show in html?

Reply:
It shows up verbatim.

<img width="190" alt="screen shot 2017-12-15 at 11 13 23 pm" src="https://user-images.githubusercontent.com/2304337/34067240-addb1f6c-e1ed-11e7-8c10-b1ecfea40256.png">